docs/tutorials/training_to_inference.md

```diff
@@ -0,0 +1,201 @@
+# DALI - from training to inference
+
+This tutorial presents the simple way of going from training to inference. We are focusing on a particular scenario: 
+how to deploy a model, when it has been trained using DALI as a preprocessing tool. An example that constitutes this 
+tutorial is the [EfficientNet](https://github.com/NVIDIA/DALI/tree/main/docs/examples/use_cases/pytorch/efficientnet) 
+from DALI repository. This tutorial is split in two parts - [Theory](#Theory) and [Practice](#Practice). Lastly, while we cover only the 
+deployment of DALI preprocessing graph, we are using handful of other NVIDIA tools useful for setting up the inference. 
+Documentation and links to those tools will be provided, but the specifics of these tools won't be explained - should 
+you like to know more details about them, please refer to the documentation.
+
+## Assumptions
+
+Since we are tackling a specific use-case in this tutorial, we will set up several assumptions:
+1. We are deploying an EfficientNet model, trained with PyTorch. The checkpoint is saved using `torch.save` and DALI 
+has been used as a preprocessing tool.
+2. The input data to this model reflects the input data used during training. To be specific - we will infer on encoded 
+JPEG files.
+3. The training script contains DALI model, implemented using DALI's `@pipeline_def` decorator. Should you have the 
+DALI pipeline implemented using other approach - please convert to `@pipeline_def` first.
+4. Triton Inference Server will be used to orchestrate the inference.
+5. The entire preprocessing part of the inference will happen on the server side, using the GPU.
+
+## Theory
+
+Unfortunately, it is not possible to introduce complete and precise algorithm for setting up the inference 
+(if it would, we'd just put together a script for it). However, the following is a highly useful "How-To" guide:
+1. Adjust a DALI Pipeline for the inference:
+    1. Make sure that the DALI pipeline you’ll be working on is the pipeline that contains the operations for the inference. Usually the **training** preprocessing and **inference** preprocessing differs - the former one contains some random operations that are used to augment the dataset. Most often the inference pipeline will match the **validation** pipeline (and not the training one).
```

Comment:
Can you link each of the theory part to the relevant `Practice` parts?

Reply:
Done